mpas_analysis/test/test_generalized_reader.py

```diff
@@ -0,0 +1,100 @@
+"""
+Unit test infrastructure for the generalized_reader.
+
+Xylar Asay-Davis
+02/15/20167
+"""
+
+import pytest
+from mpas_analysis.test import TestCase, loaddatadir
+from mpas_analysis.shared.generalized_reader.generalized_reader \
+    import open_multifile_dataset
+
+
+@pytest.mark.usefixtures("loaddatadir")
```

Comment:
There may be a better way to do this, but let's just keep this in mind for future refractors / clean up.  This is probably vestigial code on my end.